test/tests_for_db/ee2_MongoUtil_test.py

```diff
@@ -57,6 +58,39 @@ def test_init_ok(self):
         mongo_util = self.getMongoUtil()
         self.assertTrue(set(class_attri) <= set(mongo_util.__dict__.keys()))
 
+    def test_update_jobs_success(self):
+        job = get_example_job(status=Status.created.value)
+        job2 = get_example_job(status=Status.created.value)
+        job_id1 = job.save().id
+        job_id2 = job2.save().id
+        job_ids = [job_id1, job_id2]
+        scheduler_ids = [job.scheduler_id, job2.scheduler_id]
+        jobs_to_update = list(map(JobIdPair, job_ids, scheduler_ids))
+        print(jobs_to_update)
+        assert Status.created.value in [job.status, job2.status]
```

Comment:
I don't think this is doing what you want - if one of the jobs has a created status but the other doesn't this will pass

Reply:
Yea I don't know what I was thinking here, these tests are pretty bad right now, will fix em.

Reply:
:+1: